src/test/java/com/shopping_app/shoppingApp/controller/UserControllerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.shopping_app.shoppingApp.controller;
+
+import com.shopping_app.shoppingApp.model.AbstractClass.Response.ApiResponse;
+import com.shopping_app.shoppingApp.model.User.Request.UserLoginRequest;
+import com.shopping_app.shoppingApp.model.User.Request.UserUpdateRequest;
+import com.shopping_app.shoppingApp.payload.MockPayload;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.Map;
+import java.util.Objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class UserControllerTest extends AbstractController {
+
+    @Test
+    public void testUserRegister() {
+        ResponseEntity<ApiResponse> httpResponse = addUser();
+        assertEquals(200, httpResponse.getStatusCodeValue());
```

Comment:
Can you use http status codes from http class instead of hard coded numbers?

Reply:
done